src/com/maddyhome/idea/vim/command/CommandBuilder.kt

```diff
@@ -24,6 +27,9 @@ class CommandBuilder(private var currentCommandPartNode: CommandPartNode) {
   var expectedArgumentType: Argument.Type? = null
     private set
 
+  var prevExpectedArgumentType: Argument.Type? = null
+    private set
+
```

Comment:
Is this value presented to properly support some commands with digraphs? E.g. `dt<C-k>o:`.

Reply:
Yes, it is. Without it , e.g. testDeleteToDigraph will fail.